master/internal/db/postgres_experiments.go

```diff
@@ -1212,7 +1212,7 @@ func (db *PgDB) ExperimentByTrialID(id int) (*model.Experiment, error) {
 	experiment := model.Experiment{}
 	return &experiment, db.sql.QueryRowx(`
 SELECT e.id, e.state, e.config, e.model_definition, e.start_time, e.end_time,
-e.archived, e.git_remote, e.git_commit, e.git_committer, e.git_commit_date
```

Comment:
because of the duplication in these get experiment queries, and because we had a bug essentially here, not including owner id, i think i can get down with @ioga 's thought in #infra that we should consider an orm

Reply:
using an orm would disallow this

Reply:
👍🏼  I don't know if we have notes on this topic or not IIRC we had talked about it before. It'd help to see where that got stuck.